johnsonandjohnson/Bodiless-JS
# fix(gatsby-theme-bodiless): Fix revert does not refresh page

## Changes
- Updated Gatsby version to 2.19.17
- Add page reload after git revert
- Update Revert confirmation message 

## Test Instructions

- Run `npm run start`
- Make changes when in edit mode
- Click the Revert Button
- Confirm Reverting
- All the changes should be reverted.

## Related Issues
Fixes: https://github.com/johnsonandjohnson/Bodiless-JS/issues/109
